File: Web/src/app/shared/entities/survey.ts
export class Survey {
  id: number;
  name: string;
  description: string;
  surveyQuestions: SurveyQuestion[] = [];
}

export class SurveyQuestion {
  id: number;
  questionText: string;
  sortOrder: number;
  surveyQuestionAnswers: SurveyQuestionAnswer[] = [];
}

export class SurveyQuestionAnswer {
  id: number;
  answerText: string;
  sortOrder: number;
}

File: Api/Api/Data/Entities/Survey.cs
using System.Collections.Generic;

namespace Api.Data.Entities
{
    public class Survey
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public IEnumerable<SurveyQuestion> SurveyQuestions { get; set; }
    }
}
